app/routes/collections/collection/add.js

```diff
@@ -3,7 +3,7 @@ import Ember from 'ember';
 
 export default Ember.Route.extend({
 
-    panelActions: Ember.inject.service('panelActions'),
+    //panelActions: Ember.inject.service('panelActions'),
```

Comment:
can remove

Reply:
are you comfortable removing it? if so, go for it